fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -447,7 +439,7 @@ private RdfStream getTriples(final FedoraResource resource, final TripleCategory
 
     protected URI getUri(final FedoraResource resource) {
         try {
-            final String uri = translator().reverse().convert(resource).getURI();
+            final String uri = this.identifierConverter.toExternalId(resource.getId());
```

Comment:
It seems safer to use `identifierConverter()` getter... instead of the `this.identifierConverter` directly.

Reply:
will do.

Reply:
Done.